src/box/lua/schema.lua

```diff
@@ -2761,7 +2761,7 @@ space_mt.bsize = function(space)
     if s == nil then
         box.error(box.error.NO_SUCH_SPACE, space.name, 2)
     end
-    return builtin.space_bsize(s)
+    return tonumber(builtin.space_bsize(s))
```

Comment:
This is a user-visible change. I think it should be mentioned in the release notes.

Reply:
Added changelog.